runtime/fs_node.js

```diff
@@ -40,30 +41,31 @@ MlNodeDevice.prototype.exists = function(name) {
   try {
     return this.fs.existsSync(this.nm(name))?1:0;
   } catch (err) {
-    caml_raise_sys_error(err.toString());
+    // TODO: Is it correct to always return a boolean, even on error?
+    return 0;
```

Comment:
@hhugo should this **ever** raise or would it just always return a boolean?

Reply:
Always returning a boolean seems fine. I don't think the native ocaml runtime would ever raise.